packages/web/src/services/analytics/index.ts

```diff
@@ -25,14 +19,7 @@ const didInit = new Promise((resolve, reject) => {
 export const init = async () => {
   try {
     await remoteConfigInstance.waitForRemoteConfig()
-    const useAmplitude = remoteConfigInstance.getRemoteVar(
```

Comment:
Funny this was a check. Does this help us run analytics earlier in the process?

Reply:
yeah probably, but pretty negligible. We switched from segment to amplitude and never cleaned this up, speaking of feature flag cleanups :X